src/helm/env.d/production/secrets.enc.yaml

```diff
@@ -0,0 +1 @@
+../../../../secrets/numerique-gouv/impress/env/produciton/secrets.enc.yaml
```

Comment:
```suggestion
../../../../secrets/numerique-gouv/impress/env/production/secrets.enc.yaml
```

Reply:
The error is on the secrets repository. I will fix the typo on both !